.travis.yml

```diff
@@ -66,10 +66,12 @@ addons:
     - clang-format
     - clang-omp
     - ccache
+    - python
+    - pip
 install: |
     if [ $TRAVIS_OS_NAME == linux ]; then bash tools/ci/check-commit.sh; fi
 script: |
-    mkdir -p build && cd build && CC=/usr/bin/clang CXX=/usr/bin/clang++ cmake .. && \
+    pip install requests && mkdir -p build && cd build && CC=/usr/bin/clang CXX=/usr/bin/clang++ cmake .. && \
```

Comment:
move this install to ci/ci_check.sh, here only compile

Reply:
done.